tests/commands/test_record_tests_minitest.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase, mock
+from nose.tools import eq_, ok_
+from click.testing import CliRunner
+
+import os
+import json
+from pathlib import Path
+import gzip
+
+from launchable.__main__ import main
+from launchable.utils.gzipgen import compress
+
+class MinitestTest(TestCase):
+    launchable_token='v1:launchableinc/mothership:auth-token-sample'
+    session = '/intake/organizations/launchableinc/workspaces/mothership/builds/123/test_sessions/16'
+    test_files_dir = Path(__file__).parent.joinpath('../data/minitest/').resolve()
+    result_file_path = test_files_dir.joinpath('record_test_result.json')
+
+    def setUp(self):
+        os.environ['LAUNCHABLE_TOKEN'] = self.launchable_token
+
+    @mock.patch('requests.request')
```

Comment:
You need to teach me what black magic happens behind this!!

Reply:
Black magic is black magic🪄 https://docs.python.org/3/library/unittest.mock.html